src/main/java/com/exasol/releasedroid/maven/MavenPluginValidator.java

```diff
@@ -56,16 +56,18 @@ public Report validatePluginVersionEqualOrGreater(final String pluginName, final
 
     private Report validatePluginVersion(final String pluginName, final String expectedVersion) {
         final Report report = Report.validationReport();
-        final String actualVersion = this.plugins.get(pluginName).getVersion();
-        if (compareSemanticVersion(expectedVersion, actualVersion)) {
+        final MavenPlugin plugin = this.plugins.get(pluginName);
+        final String actualVersion = plugin.getVersion();
+        if (plugin.hasVersion() && compareSemanticVersion(expectedVersion, actualVersion)) {
```

Comment:
you check the version here after you run getVersion(). I would expect that if it's null the get will fail somehow. The tests prove however, that it's not the case...

Reply:
I've inline the variable to avoid confusions